lib/tasks/one_time_use/recreate_sms_jobs.rake

```diff
@@ -0,0 +1,60 @@
+namespace :one_time_use do
+  desc <<-DESC
+This task will recreate all the scheduled sms jobs, both the one-time ones, as well as periodic (every day, every month etc). \
+The tolerance for assuming it's the same job is 6 hours \
```

Comment:
Why 6? It can be tricky... For example, you have the first period "early morning (7:00-9:00)" between them you have only 3 hours so sometimes your check can return an incorrect value

Reply:
☝️

Reply:
Yeah, I guess the edge case is 5 hours (`["afternoon", 12, 17]`). However, each SendSmsJob stores the `id` of the user session, and as it's not possible to include multiple Phone Text Messaging questions in one session, I don't think it's that huge of an issue

Reply:
But you weren't sure if these SMS-es belong to the same user session. If the session has multiple fills set on true it can is possible. And 'user_session_id` parameter was added relatively recently and not all job has it